gwswlib/threedi.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
+import logging
+
+from gwswlib.sql_models.constants import Constants
+
+# from gwswlib.sql_models.constants import Constants
+
+
+logger = logging.getLogger(__name__)
+
+
+MANHOLE_SHAPE_MAPPING = {
+    "RND": Constants.MANHOLE_SHAPE_ROUND,
+    "RHK": Constants.MANHOLE_SHAPE_RECTANGLE,
+}
+
+# skipping "VRL"
+CALCULATION_TYPE_MAPPING = {
+    "KNV": Constants.CALCULATION_TYPE_ISOLATED,
+    "RES": Constants.CALCULATION_TYPE_CONNECTED,
+}
+
+MATERIAL_MAPPING = {
+    None: None,
+    "": None,
+    "BET": Constants.MATERIAL_TYPE_CONCRETE,
+    "PVC": Constants.MATERIAL_TYPE_PVC,
+    "GRE": Constants.MATERIAL_TYPE_STONEWARE,
+    "GIJ": Constants.MATERIAL_TYPE_CAST_IRON,
+    "MSW": Constants.MATERIAL_TYPE_BRICKWORK,
+    "HPE": Constants.MATERIAL_TYPE_HPE,
+    "PIJ": Constants.MATERIAL_TYPE_SHEET_IRON,
+    "STL": Constants.MATERIAL_TYPE_STEEL,
+}
+
+# skipping "CMP" and "ITP"
+MANHOLE_INDICATOR_MAPPING = {
+    "INS": Constants.MANHOLE_INDICATOR_MANHOLE,
+    "UIT": Constants.MANHOLE_INDICATOR_OUTLET,
+}
+
+
+class Threedi:
+    def __init__(self):
+        pass
+
+    def __repr__(self):
+        pass
+
+    def import_hydx(self, hydx):
+        self.connection_nodes = []
+        self.manholes = []
+
+        for connection_node in hydx.connection_nodes:
+            self.parse_connection_node(connection_node)
+
+        print(self.connection_nodes[1])
+        print(self.manholes[1])
+
+    def parse_connection_node(self, hydx_connection_node):
+        """ parse hydx.connection_node into threedi.connection_node and threedi.manhole
+
+        :param connection_node:
```

Comment:
Met dit soort zaken moet je oppassen: de parameter is nog niet gedocumenteerd ("dat doe ik later wel"), maar de parameternaam is ook nog eens incorrect.

Dan kan je het beter gelijk invullen. Of (en dat is ook vaak goed genoeg) in hoofdletters er expliciet `TODO` achter zetten.